pkg/bundle/source_test.go

```diff
@@ -499,7 +512,7 @@ func TestBundlesDeduplication(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 
-			resultBundle, err := deduplicateBundles(test.bundle)
+			resultBundle, err := deduplicateAndSortBundles(test.bundle)
 
 			assert.Nil(t, err)
 
```

Comment:
Just below, I've noticed we don't check that the deduplicated bundle uses the same order as the expected bundle:

```go
assert.ElementsMatch(t, test.testBundle, resultBundle)
```

Could we change this to check the order?

```go
assert.Equal(t, test.testBundle, resultBundle)
```

Reply:
Yeah I noticed that. That is why I put my new test in the test suite above that which actually asserts the order.

Reply:
I agree with @maelvls that we should assert the order here as well - if it does require too many additional changes.

Reply:
ok i changed that test as well